tests/lib/switch_test_setup_helpers_test.cc

```diff
@@ -36,12 +36,19 @@ constexpr uint32_t kActionId = 16777217;
 // Any constant is fine here.
 constexpr uint32_t kDeviceId = 100;
 
+<<<<<<< HEAD
```

Comment:
@divyagayathri-hcl 
@VSuryaprasad-HCL 
Merge comments got into the file here and below, pls fix.

Reply:
Kishan,
Merge Conflicts are resolved and file is updated now.